### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by cuteman18**
### POPUP

[Demo](https://cuteman18.github.io/popup/)
[Code base](https://github.com/cuteman18/popup/)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by cuteman18** [comment]
This is a substitution of #412.  (#412 was removed because of incorrect branch & directory names)

**Comment by cuteman18** [comment]
@lenadgit can you please review this PR?  It is a substitution of my PR #213 which you reviewed a few week ago.  You wrote then that once I would fix what you had commented, you would merge it.

File: submissions/cuteman18/popup/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+              <li class="icons"><a href="#" class="ref2"><span class="icon"><img class="imgStyle" src="images/files.png" alt="files"/></span><span class="iconText">files</span></a></li>
+              <li class="icons"><a href="#" class="ref2"><span class="icon"><img class="imgStyle" src="images/contacts.png" alt="contacts"/></span><span class="iconText">contacts</span></a></li>
+              <li class="icons"><a href="#" class="ref2"><span class="icon"><img class="imgStyle" src="images/photos.png" alt="photos"/></span><span class="iconText">photo</span></a></li>
+              <li class="icons"><a href="#" class="ref2"><span class="icon"><img class="imgStyle" src="images/videos.png" alt="videos"/></span><span class="iconText">video</span></a></li>
+              <li class="icons"><a href="#" class="ref2"><span class="icon"><img class="imgStyle" src="images/search.png" alt="search"/></span><span class="iconText">search</span></a></li>
+              <hr>
+              <input type="checkbox" class="button" id="extra"/><label for="extra" class="more">more</label>
```
Comment (A-Ostrovnyy) [inline submissions/cuteman18/popup/index.html:30]: This button still without styles on focus state.
I wrote about this in [PR213](https://github.com/kottans/frontend-2021-homeworks/pull/213) and [PR412](https://github.com/kottans/frontend-2021-homeworks/pull/412)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good Work!
PR title must contain a task name.

File: submissions/cuteman18/popup/index.html (hunk 1 above)
Comment (cuteman18) [reply submissions/cuteman18/popup/index.html:30]: @A-Ostrovnyy  Hello Alexander, sorry I thought you were asking about adding more focus on icons.    I added focus to the button (lines 85-88 in app.css), please review.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**